src/main/scala/tech/cryptonomic/conseil/metadata/MetadataService.scala

```diff
@@ -17,6 +17,10 @@ class MetadataService(config: PlatformsConfiguration,
                       cacheConfiguration: AttributeValuesCacheConfiguration,
                       tezosPlatformDiscoveryOperations: TezosPlatformDiscoveryOperations) {
 
+  // checks if attribute is valid
+  def isAttributeValid(tableName: String, columnName: String): Future[Boolean] =
```

Comment:
You're mixing levels here. In method name is `attribute` but in parameters list, you refer to `tableName` and `columnName`.

Reply:
your're right, changed